### PR TITLE
add/update advance rosa workflow

### PR DESCRIPTION
**Pull request by yasun1**

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-hcp-3zones-data-path-9nodes-periodic periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-stable-aws-rosa-sts-p2-f7 periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-hypershift-guest-stage-full-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-hcp-3zones-data-path-9nodes-periodic periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-hypershift-guest-stage-full-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-hcp-3zones-data-path-9nodes-periodic

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-hypershift-guest-stage-full-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-hypershift-guest-stage-full-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-rosa-aws-non-sts-advanced-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-rosa-aws-sts-advanced-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-rosa-aws-sts-hypershift-advanced-f3

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-rosa-aws-non-sts-advanced-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-rosa-aws-sts-advanced-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-rosa-aws-sts-hypershift-advanced-f3

**Comment by yasun1** [comment]
@xueli181114 @yuwang-RH Please help review.

**Comment by yasun1** [comment]
@liangxia Please help review. Thanks.

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-rosa-aws-non-sts-advanced-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-rosa-aws-sts-advanced-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-rosa-aws-sts-hypershift-advanced-f3

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml [hunk 1]
```diff
@@ -31,6 +31,30 @@ tests:
...
+    workflow: rosa-aws-non-sts-advanced
+- as: rosa-aws-sts-advanced-f3
+  cron: 41 2 2,5,7,11,14,17,20,23,26,29 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: stable
+      ROSA_LOGIN_ENV: staging
+    workflow: rosa-aws-sts-advanced
+- as: rosa-aws-sts-hypershift-advanced-f3
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:50]: Could you explain a little bit about why setting all those jobs to f3 ?   CC @jianlinliu

Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:50]: These jobs could be used for the daily testing on the cluster provisioning with the rosa cli e2e in the future. Now we enable the day 1 validation testing.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:50]: Thanks for the explanations.  Waiting for an ack from @jianlinliu as we are exceeding the budgets.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:50]: ack

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml [hunk 2]
```diff
@@ -31,6 +31,30 @@ tests:
...
     - ref: openshift-extended-rosacli-tests
     workflow: rosa-aws-sts-rosacli-e2e
+- as: rosa-aws-non-sts-advanced-f3
+  cron: 23 5 1,4,7,10,13,16,19,22,24,28 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: stable
+      ROSA_LOGIN_ENV: staging
+    workflow: rosa-aws-non-sts-advanced
```
[the remark this answers is not in the document]
Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:41]: Yes, it is expected. For now, the build process with kind of filters for the rosa cli test cases under openshift-test-private is not ready, and currently, we just want to do day 1 validation, so I removed the `test` part.

[the remark this answers is not in the document]
Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:41]: Ack

**Pull request merged**